sinks/hawkular/driver.go

```diff
@@ -0,0 +1,236 @@
+package hawkular
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/GoogleCloudPlatform/heapster/extpoints"
+	"github.com/golang/glog"
+	"github.com/hawkular/hawkular-client-go/metrics"
+
+	sink_api "github.com/GoogleCloudPlatform/heapster/sinks/api"
+	kube_api "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+)
+
+const (
+	units_tag string = "units"
```

Comment:
nit: Take a look at the go style guide [here](https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Mixed_Caps) and update the code.